### PR TITLE
clickhouse: update the clickhouse homepage to reflect the recent info architecture changes

### PR DESCRIPTION
**Pull request by wojcik-dorota**
Changes as planned in the ClickHouse doc-as-product MVP in Mirro https://miro.com/app/board/uXjVPK1O1is=/?moveToWidget=3458764536732657883&cot=14

- Added new tiles: `Quickstart`, `Overview`, `What's new`
- To avoid duplicating content, I removed introductory paragraphs - this info has already been covered in two new articles in the `Overview` section (Feature overview & Architecture overview)
- Some minor editorial improvements to names of sections for consistency and better readability

https://aiven.atlassian.net/browse/DOC-67

### CODE REVIEW

**Review by lornajane — CHANGES_REQUESTED** [review]
A few minor comments that we should revisit

File: docs/products/clickhouse.rst [hunk 1]
```diff
@@ -1,60 +1,42 @@
...
-===========
+Aiven for ClickHouse®
+=====================
 
-What is Aiven for ClickHouse®?
-------------------------------
+Fully-managed distributed columnar database based on `open source ClickHouse <https://clickhouse.com/>`_ - a fast resource-effective solution tailored for data warehouse and generation of real-time analytical data reports using advanced SQL queries.
 
-Aiven for ClickHouse® is powered by `ClickHouse <https://clickhouse.com/>`_, a highly scalable, open source database that uses a column-oriented structure. ClickHouse is designed for online analytical processing (OLAP) applications, and is an ideal tool for applications such as web analytics, or complex data reporting.
+.. grid:: 2
```
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/clickhouse.rst:6]: Thanks for the clarification! Reverting to the original grid definition...

File: docs/products/clickhouse.rst [hunk 2]
```diff
@@ -1,60 +1,42 @@
...
-* Our :doc:`/docs/products/clickhouse/getting-started` guide is a good way to get hands on with your first project.
+        Look up parameters, metrics, or formats supported by Aiven for ClickHouse.
 
-* Follow :doc:`a guide to add a sample dataset </docs/products/clickhouse/howto/load-dataset>` to your service.
+    .. grid-item-card:: `WHAT'S NEW <https://aiven.io/changelog>`_
+        :shadow: md
+        :margin: 2 2 0 0
 
-* Find out :doc:`how to use the web-based query editor </docs/products/clickhouse/howto/query-databases>`, that comes as part of the ClickHouse service.
+        Check out new features or latest improvements introduced to Aiven for ClickHouse.
```
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/clickhouse.rst:42]: We're in the transition of the changelog into Sanity and there's gonna be a filtering mechanism available for the feeds some time soon. For now, I'll reword the description as you advised.

**Review by lornajane — APPROVED** [review]
Very neat, very happy to approve and merge :)

**Pull request merged**